### PR TITLE
Neo4j connection

### PR DESCRIPTION
**Pull request by diversifiedmechanic**
And a lot more.  I'm really sorry about this.

### CODE REVIEW

File: database/neo4j/db.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const neo4j = require('neo4j-driver');
+require('dotenv').config();
+
+module.exports.driver = neo4j.driver(
+  process.env.NEO_URL,
+  neo4j.auth.basic(process.env.NEO_USER, process.env.NEO_PW),
+);
+
+//  = driver.session();
```
Comment (naszzz) [inline database/neo4j/db.js]: lgtm

File: database/neo4j/index.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            rev.noise AS noise,
+            rev.overall AS overall,
+            rev.recommend AS recommend,
+            rev.service AS service,
+            r2.date AS dineDate,
+            reviewer.reviewerId AS reviewerId,
+            reviewer.color AS color,
+            reviewer.firstName AS firstName,
+            reviewer.lastName AS lastName,
+            reviewer.vip AS vip
```
Comment (naszzz) [inline database/neo4j/index.js]: looks good. i mean it's unfortunate you couldn't pass the order by variable, but still. :)

File: database/postgres/index.js [hunk 3]
```diff
@@ -1,15 +1,44 @@
...
+          reviews.id,
+          reviewers.id,
+          reviewers.firstname,
+          reviewers.lastname,
+          reviewers.color,
+          reviewers.vip
+        FROM
+          restaurants,
+          reviews,
+          reviewers
```
Comment (naszzz) [inline database/postgres/index.js]: looks good. honestly tried to do something similar and still wasn't working out for me. nice to see you got it in one query .

File: k6script.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  rps: 1000
+};
+
+export default function() {
+  let res = http.get("http://localhost:3300/5000");
+  check(res, {
+    "status was 200": (r) => r.status == 200,
+    "transaction time OK": (r) => r.timings.duration < 2000
+  });
+};
```
Comment (naszzz) [inline k6script.js]: lgtm.

File: database/postgres/index.js (hunk 3 above)
Comment (diversifiedmechanic) [reply database/postgres/index.js]: I found something that may help me and I"ll try to come back and use it if I have time.

**Pull request merged**